examples/tf-azurerm-computegroup/main.tf

```diff
@@ -0,0 +1,122 @@
+resource "azurerm_resource_group" "rg" {
+  name     = "${var.resource_group_name}"
+  location = "${var.location}"
+  tags     = "${var.tags}"
+}
+
+resource "azurerm_virtual_network" "vnet" {
+  name                = "acctvn"
+  address_space       = ["10.0.0.0/16"]
+  location            = "${var.location}"
+  resource_group_name = "${azurerm_resource_group.rg.name}"
+}
+
+resource "azurerm_subnet" "subnet1" {
+  name                 = "acctsub"
+  resource_group_name  = "${azurerm_resource_group.rg.name}"
+  virtual_network_name = "${azurerm_virtual_network.vnet.name}"
+  address_prefix       = "10.0.1.0/24"
+}
+
+resource "azurerm_public_ip" "publicip" {
+  name                         = "test"
+  location                     = "${var.location}"
+  resource_group_name          = "${azurerm_resource_group.rg.name}"
+  public_ip_address_allocation = "static"
+  domain_name_label            = "${azurerm_resource_group.rg.name}"
+  tags                         = "${var.tags}"
+}
+
+resource "azurerm_lb" "loadbalancer" {
+  name                = "loadbalancer"
+  location            = "${var.location}"
+  resource_group_name = "${azurerm_resource_group.rg.name}"
+  tags                = "${var.tags}"
+
+  frontend_ip_configuration {
+    name                 = "PublicIPAddress"
+    public_ip_address_id = "${azurerm_public_ip.publicip.id}"
```

Comment:
_minor/not a blocker_ for the Terraform resource name (in the context of the PIP and the RG) - you may wish to use something more descriptive, i.e. "azurerm_resource_group.vmss" rather than `rg` and `publicip`, as it can help make the console output clearer :)